SEO excerpts: Use part 3 (#3832)
timescale/docs: use-timescale/data-tiering/enabling-data-tiering.md

---
title: Manage tiering

> excerpt: Enable tiered storage for your data in Timescale Console. With automated data tiering policies, you get a set it and forget it tool to cut storage costs

Reviewer: ```suggestion
excerpt: Enable tiered storage for your data in Timescale Console. Automated data tiering policies give you a `set it and forget it` tool to cut storage costs
```
